src/components/RelocateBoxForm/RelocateBoxForm.jsx

```diff
@@ -12,14 +12,14 @@ import {
   Input,
```

Comment:
"Support for jpeg, jpg, png" should be centered.

Reply:
done